src/onegov/org/cronjobs.py

```diff
@@ -47,6 +48,8 @@ def hourly_maintenance_tasks(request):
     publish_files(request)
     reindex_published_models(request)
     send_scheduled_newsletter(request)
+    # for testing
+    # archive_old_tickets(request)
```

Comment:
Is the idea here that we don't enable this part of the cronjob until a later date?

Reply:
This needs to be removed, was just for local testing. I have configured both jobs to run in the night (e.g. once every 24 hours)